docs/001_develop/03_client-capabilities/008_interaction/checkbox.mdx

```diff
@@ -50,16 +50,16 @@ Usage
   template: html`
     <rapid-checkbox
       ?disabled="${(x) => x.disabled}"
-      ?checked="${sync((x) => x.isSelected, 'boolean')}"
-      @change="${(x) => x.checkChanged()}"
+      @change="${(x, ctx) => x.checkChanged(ctx.event)}"
```

Comment:
Can we leave `sync` in this example please. If you want to show off the event version you can leave that too and comment on the funciton saying it's a choice